### PR TITLE
Ci pipeline

### PR DESCRIPTION
**Pull request by iorubs**
One of the tests is failing, seems like the files with '.' like the .dockerignore are being ignored.

The builder in me wants to disable it and just have it merged to have a package in npm but the DevOps in me doesn't like that at all.

### CODE REVIEW

File: generator/template/default/goss.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+process:
```
Comment (DaniJG) [inline generator/template/default/goss.yaml]: Should this file be optional? Might surprise many (like me) who dont know what goss is

File: README.md [hunk 2]
```diff
@@ -1,2 +1,71 @@
-# vue-cli-plugin-vuedock
-Docker plugin for the VueCLI
+# VueDock
+> Docker plugin for the Vue CLI 3
+
+[![Build Status](https://travis-ci.org/kaizendorks/vue-cli-plugin-vuedock.svg?branch=master)](https://travis-ci.org/kaizendorks/vue-cli-plugin-vuedock)
+
```
Comment (DaniJG) [inline README.md]:
```suggestion

Generates docker and compose files for both development and hosting, allowing:
- running the development server
- running unit tests
- running nightwatch e2e tests
- hosting built application with nginx container

## Installation in a Vue project
Simply run from the project root:
```vue add vuedock```

For more information, see: https://cli.vuejs.org/guide/plugins-and-presets.html#installing-plugins-in-an-existing-project
```

File: README.md [hunk 3]
```diff
@@ -1,2 +1,71 @@
...
+## Instructions
```
Comment (DaniJG) [inline README.md]: Should we change this title to `Contributing` ?

File: README.md [hunk 4]
```diff
@@ -1,2 +1,71 @@
...
+  && vue invoke vue-cli-plugin-vuedock
+```
+
+#### CI
+
+``` bash
+# Build image
+docker build -t vuedock .
+
+# Dgoss Tests (quick smoke test to check if docker files were generated)
```
Comment (DaniJG) [inline README.md]: Could you add a one liner on what dgoss is, a link to its documentation and what we use it for?

File: generator/template/default/goss.yaml (hunk 1 above)
Comment (iorubs) [reply generator/template/default/goss.yaml]: Yeah, good point.

File: README.md (hunk 2 above)
Comment (iorubs) [reply README.md]: Ah never knew you could do this in github, very cool.

File: README.md (hunk 3 above)
Comment (iorubs) [reply README.md]: Yeah I actually added a todo at the end:  "Update readme and move this instructions somewhere else".

Note sure where a good place would be tho.

File: README.md (hunk 4 above)
Comment (iorubs) [reply README.md]: Done.

File: generator/template/default/goss.yaml (hunk 1 above)
Comment (iorubs) [reply generator/template/default/goss.yaml]: done.

File: README.md (hunk 3 above)
Comment (iorubs) [reply README.md]: Left it in the readme but added "Contributing", we can move somewhere else after if needed.

**Pull request merged**